experimental/segmenter/src/line.rs

```diff
@@ -621,6 +638,7 @@ pub struct LineBreakIterator<'l, 's, Y: LineBreakType<'l, 's> + ?Sized> {
     options: &'l LineBreakOptions,
     dictionary: &'l Dictionary,
     lstm: &'l LstmPayloads,
+    grapheme: Option<&'l GraphemeClusterSegmenter>,
```

Comment:
Is it possible to make this be a `Option<GraphemeClusterBreakIterator<'l>>`? It should be more efficient that way.

Reply:
I think we need the segmenter type in `DictionarySegmenter` or `LstmSegmenter` in order to create dictionary/lstm break iterator for a substring of the original input string. I made this assessment by reading `complex_language_segment_str`.

Reply:
My concern has still not been addressed. This really is something I want to see before merging to components. However, it can be done in a follow-up PR.

Reply:
I added this as a to-do item in #2259:

>  Make LineBreakIterator borrow the data from inside of GraphameClusterSegmenter instead of borrowing the whole segmenter